test/integration/services/data/test-insert-offender-manager.js

```diff
@@ -0,0 +1,30 @@
+const expect = require('chai').expect
+const knexConfig = require('../../../../knexfile').development
+const knex = require('knex')(knexConfig)
+const config = require('../../../../config')
+const insertWorkloadOwner = require('../../../../app/services/data/insert-offender-manager')
+
+describe('app/services/data/insert-offender-manager', function () {
+  const tableName = `${config.DB_APP_SCHEMA}.offender_manager`
+  it('should insert a new offender manager record', function () {
+    insertWorkloadOwner().then(function (offenderManagerId) {
```

Comment:
the `insertWorkloadOwner` method expects a domain object to insert as a parameter.